ghost/core/core/bridge.js

```
@@ -38,6 +38,17 @@ class Bridge {
         });
+        events.on('settings.labs.edited', (model) => {
+            debug('theme setting changed');
+            if (model.get('value').themeTranslation) {
+                debug('theme setting changed to true');
+            } else {
+                debug('theme setting changed to false');
+            }
+            const locale = settingsCache.get('locale');
+            this.getActiveTheme().initI18n({locale});
+
+        });
```

ESLint

[error] Block must not be padded by blank lines.

(padded-blocks)